templates/zerver/tests/markdown/test_tabbed_sections_missing_tabs.md

```diff
@@ -0,0 +1,11 @@
+# Heading
+
+{start_tabs}
+{tab|ios}
+iOS instructions
+
+{tab|minix}
+
+Minix instructions
```

Comment:
I would consider phrasing this in a way that makes clear the intent of this file. "Minix instructions.  We expect an exception because the `minix` tab doesn't have a defined name."

Reply:
Ack